cmake_templates/AndroidManifest.xml.in

```diff
@@ -65,40 +61,36 @@
             <meta-data 
                 android:name="android.app.arguments" 
                 android:value="-- %%INSERT_APP_ARGUMENTS%% --" />
-
-            <!-- Background running -->
-            <!-- Warning: changing this value to true may cause unexpected crashes if the
-                          application still try to draw after
-                          "applicationStateChanged(Qt::ApplicationSuspended)"
-                          signal is sent! -->
-            <meta-data android:name="android.app.background_running" android:value="false"/>
-            <!-- Background running -->
-
-            <!-- extract android style -->
-            <!-- available android:values :
-                * default - In most cases this will be the same as "full", but it can also be something else if needed, e.g., for compatibility reasons
-                * full - useful QWidget & Quick Controls 1 apps
-                * minimal - useful for Quick Controls 2 apps, it is much faster than "full"
-                * none - useful for apps that don't use any of the above Qt modules
-                -->
-            <meta-data 
```

Comment:
are you confident we can remove this one?

Reply:
Here I am more confident as I saw this commit: https://github.com/qt/qtbase/commit/ee4f91fa50fa470b4891318750bd7aad5e508124